clients/geth/specular/sbin/import_accounts.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+SBIN=`dirname $0`
+SBIN="`cd "$SBIN"; pwd`"
+. $SBIN/configure.sh
+cd $DATA_DIR
```

Comment:
are all these scripts supposed to be run w/ `source` (`. my_script.sh`) or `./my_script.sh`? if the former, i’d avoid `cd` here and just use the absolute paths below (i.e. `$GETH_SPECULAR_DIR/build/bin/geth` etc)

same question for the other scripts